UCSRest/src/test/java/it/cnr/iit/ucsrest/coverage/ContextHandlerCoverageTests.java

```diff
@@ -56,45 +58,38 @@ private void init() throws URISyntaxException, IOException, JAXBException {
     }
 
     @Test( expected = PreconditionException.class )
-    public void contextHandlerTryAccessShouldFail() throws PreconditionException {
+    public void contextHandlerTryAccessShouldFail() throws PreconditionException, PolicyException, RequestException {
```

Comment:
again my view of exceptions in test methods as before

Reply:
yeah I'll have to refactor these tests someday :D